### ISSUE TITLE
GRPC reflection does not show all services

### ISSUE THREAD

**Issue by aditanase** [a85/POSTMan-Chrome-Extension#10569]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request

### Is your feature request related to a problem?

Using 9.9.0 I am trying to load service definition via reflection and I am only seeing 2 out of ~20 services. Tested against GrpcUI to make sure the server is properly configured.

![image](https://user-images.githubusercontent.com/2746097/149496269-9950ee25-567d-4cec-88ef-944434f73ee2.png)

![image](https://user-images.githubusercontent.com/2746097/149496232-d2af2450-22ec-4a72-af9c-f0a44d5d60ba.png)

### Describe the solution you'd like

Server reflection should work correctly.

Alternatively, offer an option to upload the descriptor file, supported by all other tools (grpcurl, grpcui, google cloud endpoints etc).

### Describe alternatives you've considered

Loading protos individually, which breaks because of #10547

**Comment by aditanase** [comment]
Apologies, tagged this as feature,  it's a bug report, I can't edit the labels.

**Comment by appurva21** [comment]
Hey @aditanase, thank you for the report!

Is this server publicly accessible? If not, will it be possible for you to share a scaled-down version of the server which we can use to reproduce the issue at our end?

Secondly, just to confirm, can you please check if the Auth and Certificates in Postman are configured correctly in accordance with your server requirements?

**Comment by Liloupar** [comment]
same issue

**Comment by aditanase** [comment]
@appurva21 Thanks. Unfortunately this is a private API, it would take a bit of work to scale down / deploy it somewhere. There is no authentication, local host with plain text.

What I did try though is to comment out some of the services served by the main Grpc server, and this time some of the missing ones appeared (but still not all), if that helps.

**Comment by AnteWall** [comment]
I can confirm I have the same issues with our private API:s that is running multiple services on one server

**Comment by Breefield** [comment]
Can confirm, this issue is coming up as a sticking point for us.

**Comment by codenirvana** [comment]
Hey everyone, can someone share a scaled-down server (or an example Protobuf) that we can use to reproduce this issue?

**Comment by pc2459** [comment]
I'll whip up a demo server this weekend, but all it really takes is adding a second service, however minimal, to any gRPC server.

**Comment by Breefield** [comment]
Oh, this works in Wombat!

**Comment by aditanase** [comment]
I managed to reproduce with 2 very simple service definitions:

`greeter.proto`
```proto
syntax = "proto3";
package example.protobuf.v1alpha2;

option java_multiple_files = true;
option java_outer_classname = "GreeterProto";
option java_package = "com.example.protobuf.v1alpha2";

import "google/api/annotations.proto";

// The greeting service definition.
service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {
    option (google.api.http) = { get: "/v1alpha2/greeter/hello" };
  }
}

// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloReply {
  string message = 1;
}

```

`greeter_v2.proto`
```proto

syntax = "proto3";
package example.protobuf.v1beta1;

option java_multiple_files = true;
option java_outer_classname = "GreeterProto";
option java_package = "com.example.protobuf.v1beta1";

import "google/api/annotations.proto";

// The greeting service definition.
service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {
    option (google.api.http) = { get: "/v1beta1/greeter/hello" };
  }
}

// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloReply {
  string message = 1;
}
```

Nothing fancy about the server
```java
    server =
        ServerBuilder.forPort(port)
            .addService(addInterceptors(new GreeterService()))
            .addService(addInterceptors(new GreeterServiceV2()))
            .addService(ProtoReflectionService.newInstance())
            .build();
```

Top of mind this looks like a conflict on method names, maybe they are not scoped correctly by package?

**Comment by codenirvana** [comment]
@aditanase Thanks for sharing this, identified the issue in resolving package names.

We'll fix this as soon as possible and update this thread.

**Comment by aditanase** [comment]
@codenirvana seems to be included in the latest version? Our services are appearing correctly in v9.16.1

**Comment by codenirvana** [comment]
@aditanase Yes.

This is fixed in v9.16.1, thanks for the confirmation!
